### DISCUSSION TITLE
Broke normal work

### DISCUSSION THREAD

**Discussion by jaka1-newbie** [ImageMagick/ImageMagick#3255] [Help]
magick.exe convert -size 100x277  -set colorspace sRGB  xc:#F1FBFF basic.jpg

convert: unable to open image 'xc:#F1FBFF': No such file or directory @ error/blob.c/OpenBlob/3539.
convert: no decode delegate for this image format `' @ error/constitute.c/ReadImage/562.

Where to dig?
It is worked normal....

**Comment by jaka1-newbie** [comment]
1 PC worked normal - another error
Where error - already reinstal windows 7

**Reply by dlemstra** [reply]
Are you using the same versions on both machines?

**Reply by jaka1-newbie** [reply]
ImageMagick-7.0.10-Q16-HDRI and ImageMagick-7.0.11-Q16-HDRI
it is so big difference?

**Reply by jaka1-newbie** [reply]
Thank God! The installation file 7.0.10 was saved on the second PC -
 installed and now working. 

BUT it is wrong when everything is so broken within the same version 7.0.10 - 7.0.11......
